ci/docker-compose-direct-model-sgx-build.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  avalon-direct-model-sgx-tests:
+    image: avalon-shell:${ISOLATION_ID}
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+
+  avalon-enclave-manager:
+    image: avalon-sgx-enclave-manager:${ISOLATION_ID}
+    build:
```

Comment:
Is only 2 containers required. Couldn't understand why this file is named as 
sgx-build as it doesn't seem to build all required Avalon components

Reply:
If you see Jenkinsfile-sgx-hw in this PR, I am building avalon components like:
docker-compose -f ci/docker-compose-direct-model-build.yaml -f ci/docker-compose-direct-model-sgx-build.yaml build